src/vdaf/prio3.rs

```diff
@@ -12,13 +12,7 @@
 //!
 //! Prio3 is a transformation of a Fully Linear Proof (FLP) system [[draft-irtf-cfrg-vdaf-01]] into
 //! a VDAF. The base type, [`Prio3`], supports a wide variety of aggregation functions, some of
-//! which are instantiated here:
-//!
-//! - [`Prio3Aes128Count`] for aggregating a counter (*)
-//! - [`Prio3Aes128CountVec`] for aggregating a vector of counters
-//! - [`Prio3Aes128Sum`] for copmputing the sum of integers (*)
-//! - [`Prio3Aes128Histogram`] for estimating a distribution via a histogram (*)
-//!
```

Comment:
Since PrgAes128 is in the default feature set, I don't think this comment should be reverted.